CHANGES.txt

```diff
@@ -1,3 +1,10 @@
+v3.8.0
```

Comment:
Next version is 3.7.0. Please add you nice detailed line to the 3.7.0 section? ;-) Unless we want to wait for the next release?